main_logic/tts_client.py

```
@@ -1247,6 +1478,11 @@ def get_tts_worker(core_api_type='qwen', has_custom_voice=False):
         tts_config = cm.get_model_api_config('tts_custom')
         # 只有当 is_custom=True（即 ENABLE_CUSTOM_API=true 且用户明确配置了自定义 TTS）时才使用本地 worker
         if tts_config.get('is_custom'):
+            base_url = tts_config.get('base_url', '')
+            # GPT-SoVITS v3：配置 http/https URL，worker 内部自动转为 ws:// 连接
```

Ruff (0.15.0)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)